frontend/src/js/Header/index.jsx

```diff
@@ -7,8 +7,12 @@ class Header extends React.Component {
   render () {
     return (
       <div>
-        <Link to='/'><h1>Amazon 2.0 Home Page</h1></Link>
-        <header style={styles}>
+        <header className='container' style={styles}>
+          <Link to='/'>
+            <div className='logo-container'>
+              <img src='../../../public/img/shamazon-logo.png'></img>
```

Comment:
@katbow if you want to run this from build replace the src with 'img/shamazong-logo.png'